### PR TITLE
Package json lint test

### PR DESCRIPTION
**Pull request by saket2403**
### Summary

Add package json lint test for all packages

Closes #

### Deployment Link

https://terra-core-deployed-pr-#.herokuapp.com/

### Additional Details

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: packages/terra-action-footer/CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,9 @@
 * Changed
   * Updated WDIO screenshots due to webpack-config-terra sass upgrade.
 
+* Added
```
Comment (benbcai) [inline packages/terra-action-footer/CHANGELOG.md:10]: Might be a merge issue this and other change logs but this should be moved to the Unreleased section.

Comment (saket2403) [reply packages/terra-action-footer/CHANGELOG.md]: Done

File: package.json [hunk 2]
```diff
@@ -27,6 +27,9 @@
   "eslintConfig": {
     "extends": "@cerner/terra"
   },
+  "package-json-lint": {
+    "extends": "@cerner/package-json-lint-config-terra"
```
Comment (pranav300) [inline package.json:31]: Can we add `projectType: module` here?

**Comment by saket2403** [comment]
Closing this PR as it is  duplicate of #3514

**Pull request closed without merging**